### DISCUSSION TITLE
Question 83: How to use the Tasks tab? [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1166] [Q&A]
Description: We have some audio files of interviews in EMu that need to be removed after five years, according to the IRB protocol. I want to set up a task to remind us to do this. I haven’t created a task before so I’d like to confirm that I’m doing it correctly (see MM IRN 2602840). Also wondering who should be notified of this in the case that none of the people currently involved in this are here in five years?

**Comment by rondlg** [comment]
Response: Do you have a google group email for your department? If so, test that and see if it alerts anyone.

- Note that in transactions emails should go to any email that is in the attached party. Let us know if that does not happen in the module you are testing.
- Embargo workflow: https://github.com/fieldmuseum/EMu-Documentation/discussions/329
& specifically [tasks tab steps](https://docs.google.com/document/d/1csugAKpqq7D22LuT_abf4YElFhEqQozbjIQHsULL4aw/edit#heading=h.yshhgy3xowks) (modify as needed for the task you’re doing)

**Comment by rondlg** [comment]
@magpiedin I think there is a ZD case for this. Can you cross reference please?

**Reply by magpiedin** [reply]
[ZD#73627](https://fieldmuseum.zendesk.com/agent/tickets/73627)
